sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -40,6 +40,9 @@
     <None Include="TestCasesForRuleFailure\**\*">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="TestCases\RestrictDeserializedTypes.JavaScriptSerializer.cs">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>

```

Comment:
Why is this needed ? Isn't this already covered by 
```
    <None Include="TestCases\**\*">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
```
above ?

Reply:
It's not needed. Was automatically added by the IDE and I forgot to remove it.